<launch>
    <arg name="fcu_url" default="udp://:14541@127.0.0.1:14581" />
    <arg name="gcs_url" default="" />
    <arg name="tgt_system" default="2" />
    <arg name="tgt_component" default="1" />
    <arg name="log_output" default="screen" />
    <arg name="fcu_protocol" default="v2.0" />
    <arg name="respawn_mavros" default="false" />

    <group ns="uav2">
        <node pkg="mavros" type="mavros_node" name="mavros2" output="$(arg log_output)">
            <param name="fcu_url" value="$(arg fcu_url)" />
            <param name="gcs_url" value="$(arg gcs_url)" />
            <param name="target_system_id" value="$(arg tgt_system)" />
            <param name="target_component_id" value="$(arg tgt_component)" />
            <param name="fcu_protocol" value="$(arg fcu_protocol)" />
            <param name="respawn" value="$(arg respawn_mavros)" />
        </node>
    </group>
</launch>

